### ISSUE TITLE
1x missing alt text https://uclalibrary.github.io/research-tips/sandbox/making-of-study-tips/

### ISSUE THREAD

**Issue by woof84** [UCLALibrary/research-tips#201]

**Comment by woof84** [comment]
fixed with updated metadata!
